src/data/services/LmsApiService.js

```diff
@@ -4,6 +4,9 @@ import { configuration } from '../../config';
 class LmsApiService {
   static baseUrl = configuration.LMS_BASE_URL;
   static pageSize = 10
+  static rolesUrl = `${LmsApiService.baseUrl}/api/enrollment/v1/roles/`;
```

Comment:
Please follow existing pattern and move this variable into the `canUserViewGradebook` function